cmd/catalyst-node/balancer/balancer.go

```diff
@@ -0,0 +1,297 @@
+package balancer
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"sync"
+
+	serfclient "github.com/hashicorp/serf/client"
+	glog "github.com/magicsong/color-glog"
+)
+
+type BalancerIface interface {
+	Start() error
+	UpdateMembers(members *[]serfclient.Member) error
```

Comment:
You could change it to `UpdateMembers(members []string) error` or `UpdateMembers(members []Member`. Then, you won't have dependency on `serfclient` in the `balancer` package.

Reply:
True, though I think we'll need our own type for them very soon. Which I guess would be... `cluster.Node`?

Reply:
Implemented in 508a8e2